.github/workflows/r.yml

```diff
@@ -90,9 +90,13 @@ jobs:
         run: |
           sudo sysctl -w kernel.core_pattern="core.%e.%p"
           ulimit -c unlimited
-          # Setting a non-default and non-probable Marquesas French Polynesia time
-          # it has both with a .45 offset and very very few people who live there.
-          archery docker run -e TZ=MART -e ARROW_R_FORCE_TESTS=${{ matrix.force-tests }} ubuntu-r
+          # TZ was set to a non-default and non-probable Marquesas French Polynesia
+          # (it has both with a .45 offset and very very few people who live there)
+          # time here. The build was failing with the following warning (probably
+          # raised to error) in them:
+          #   System has not been booted with systemd as init system (PID 1). Can't operate.
+          #   Failed to connect to bus: Host is down
+          archery docker run -e TZ="" -e ARROW_R_FORCE_TESTS=${{ matrix.force-tests }} ubuntu-r
```

Comment:
Do we still need this bit at all? I thought the new change to `skip_if_not_running_large_memory_tests()` should cover the OOMing?

Reply:
Not sure I follow. This bit is no longer needed as the failure has nothing to do with `TZ`. So I will undo this change.

Reply:
Great 👍